video/out/gpu/video.c

```diff
@@ -428,7 +424,7 @@ const struct m_sub_options gl_video_conf = {
         SCALER_OPTS("dscale", SCALER_DSCALE),
         SCALER_OPTS("cscale", SCALER_CSCALE),
         SCALER_OPTS("tscale", SCALER_TSCALE),
-        {"scaler-lut-size", OPT_INT(scaler_lut_size), M_RANGE(4, 10)},
+        {"scaler-lut-size", OPT_REMOVED("hard-coded as 256")},
```

Comment:
technically 8, not that it matters much when nobody can read the manual to know what the option did anymore